Parsers/ASimNetworkSession/Parsers/ASimNetworkSessionPaloAltoPanOS.yaml

```diff
@@ -0,0 +1,102 @@
+Parser:
+  Title: Network Session ASIM parser for Palo Alto Networks
+  Version: '0.2'
+  LastUpdated: Dec 20, 2021
+Product:
+  Name: PaloAlto
+Normalization:
+  Schema: NetworkSessions
+  Version: '0.2.1'
+References:
+  - Title: ASIM Network Session Schema
+    Link: https://aka.ms/ASimNetworkSessionDoc
+  - Title: ASIM
+    Link: https://aka.ms/AboutASIM
+  - Title: traffic log fields
+    Link: https://docs.paloaltonetworks.com/pan-os/10-1/pan-os-admin/monitoring/use-syslog-for-monitoring/syslog-field-descriptions/traffic-log-fields.html
+  - Title: Palo Alto Common Event Format Integration Guide [pdf]
+    Link: https://docs.paloaltonetworks.com/content/dam/techdocs/en_US/pdf/cef/pan-os-90-cef-configuration-guide.pdf
+Description: |
+  This ASIM parser supports normalizing Palo Alto Networks logs produced by the Microsoft Sentinel Palo Alto Networks connector to the ASIM Network Session normalized schema.
+ParserName: ASimNetworkSessionPaloAltoCEF
+EquivalentBuiltInParser: _ASim_NetworkSession_PaloAltoCEF
+ParserParams:
+  - Name: disabled
+    Type: bool
+    Default: false
+ParserQuery: |
+  let Actions=datatable(DeviceAction:string,DvcAction:string)
+    [ "reset client","Reset Source"
+    , "reset server","Reset Destination"
+    , "reset both", "Reset"
+    , "allow","Allow"
+    , "deny","Deny"
+    , "drop", "Drop"
+    , "drop ICMP", "Drop ICMP"];
+  let NWParser=(disabled:bool=false){
+  CommonSecurityLog | where not(disabled)
+  | where DeviceVendor=="Palo Alto Networks" and (Activity=="TRAFFIC")
+  | parse AdditionalExtensions with "PanOSDstPackets="DstPackets:long";PanOSsrc_packets="SrcPackets:long";start="EventStartTime:datetime";reason="*
+  | project-rename 
+        EventProduct=DeviceProduct // Not Documented
```

Comment:
Not sure why my comment got lost here: It has to be a fixed value, not the original. Use "PanOS"